librz/core/cmd/cmd_print.c

```diff
@@ -4012,8 +4013,12 @@ RZ_IPI RzCmdStatus rz_print_function_rzil_handler(RzCore *core, int argc, const
 	}
 
 	ut64 size = end - start;
+	if (size > old_blocksize) {
+		rz_core_block_size(core, size);
+	}
```

Comment:
you can always run it. there is a check within the function that does this.

Reply:
Thank you.